### PR TITLE
Cards App added brightness option

### PR DESCRIPTION
**Pull request by phrogg**
Added an option to use full brightness when showing code

### CODE REVIEW

File: apps/cards/app.js [hunk 1]
```diff
@@ -14,6 +14,22 @@
...
+let brightness;
+
+function loadBrightness() {
+  try {
+      const d = require('Storage').readJSON("setting.json", 1) || {};
+      brightness = Object.assign({
+          'brightness': 0.1
+      }, d || {});
+      return d;
+  } catch(e){
```
Comment (bobrippling) [inline apps/cards/app.js:27]: What exceptions were you seeing here? Was it from an older version of the code? `readJSON` won't through with `1` as the second argument, and nothing else throws in this code block

File: apps/cards/app.js [hunk 2]
```diff
@@ -14,6 +14,22 @@
...
 
+// get brightness
+let brightness;
+
+function loadBrightness() {
+  try {
+      const d = require('Storage').readJSON("setting.json", 1) || {};
+      brightness = Object.assign({
+          'brightness': 0.1
+      }, d || {});
```
Comment (bobrippling) [inline apps/cards/app.js:25]: To save on holding all of settings in memory, we could do:

```javascript
const settings = require('Storage').readJSON("setting.json", 1) || {};
brightness = settings.brightness || 0.1;
```

And use just `brightness` below

**Comment by bobrippling** [comment]
Thanks for the change! A few small suggestions

File: apps/cards/app.js (hunk 2 above)
Comment (phrogg) [reply apps/cards/app.js:25]: Sure sounds good to me. Did not do much too developing until now. Tips are very welcome, thanks :) You cannot however use the variable name `settings` as it is already used for the settings of the cards app itself.

File: apps/cards/app.js (hunk 1 above)
Comment (bobrippling) [reply apps/cards/app.js:27]: Not a problem, sounds like we can drop the try-catch :)

File: apps/cards/app.js (hunk 2 above)
Comment (bobrippling) [reply apps/cards/app.js:25]: Cool, that's fair, we could call it whatever you like - give me a shout if you have any trouble making the changes

**Comment by bobrippling** [comment]
Thanks for the adjustments, looks great!

**Pull request merged**